vagrant/postchecks_qat_driver.sh

```diff
@@ -16,10 +16,10 @@ echo "QAT driver validation"
 
 # Driver validation
 supported_dev="c6xx dh895xcc"
-qat_svc=$(sudo service qat_service status | grep "There is .* QAT acceleration device(s) in the system:")
+qat_svc=$(sudo systemctl status qat_service | grep "There is .* QAT acceleration device(s) in the system:")
```

Comment:
It seems that `adf_ctl status` return value is zero if acceleration devices are present in the system. If you can verify this, there is no need to parse the (fragile) service logging output.

Reply:
It returns zero if the command was executed properly:
```
[vagrant@localhost ~]$ sudo /usr/local/bin/adf_ctl status
Checking status of all devices.
There is 0 QAT acceleration device(s) in the system:
[vagrant@localhost ~]$ echo $?
0
```

Reply:
Did you try the negative case too? If both work then this might be a cleaner way to implement the check.